app/Models/Beatmapset.php

```diff
@@ -742,7 +735,7 @@ public function recentEvents()
 
     public function status()
     {
-        return static::STATES[$this->approved];
+        return array_search($this->approved, self::STATES);
```

Comment:
probably an `array_search_null` instead. Unless you like having this function returning `false`... :dancer: